ironfish/src/rpc/routes/wallet/importViewAccount.ts

```diff
@@ -0,0 +1,71 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
+import { v4 as uuid } from 'uuid'
+import * as yup from 'yup'
+import { ApiNamespace, router } from '../router'
+import { importAccount } from './utils'
+
+export type ViewAccountImport = {
+  name: string
+  viewKey: string
+  publicAddress: string
+  incomingViewKey: string
+  outgoingViewKey: string
+  version: number
+}
+
+export type ImportViewAccountRequest = {
+  account: ViewAccountImport
+  rescan?: boolean
+}
+
+export type ImportViewAccountResponse = {
+  name: string
+  isDefaultAccount: boolean
+}
```

Comment:
This is a duplicate type, we could consolidate in utils or make them equivalent.